pandas/_libs/tslibs/period.pyx

```diff
@@ -964,23 +964,8 @@ def dt64arr_to_periodarr(const int64_t[:] dtarr, int freq, tz=None):
     """
     cdef:
         int64_t[:] out
-        Py_ssize_t i, l
```

Comment:
do we even need this routine any longer? (e.g. just call localize_dt64arr_to_period in the caller)

Reply:
sure